tools/packaging/generate_debian_package.sh

```diff
@@ -70,14 +60,21 @@ cp ${VERSION_FILE} ${BINARY_TARGET_DIR}/dragonfly || cleanup "failed to copy bin
 ${BASE_PATH}/${CHANGELOG_SCRIPT} ${REPO_PATH} || cleanup "failed to generate changelog for package"
 
 MY_DIR=${PWD}
-cd ${BASE_PATH} && dpkg-buildpackage --build=binary || cleanup "failed to generate the package"
+cd ${BASE_PATH}
+dpkg-buildpackage --build=binary || cleanup "failed to generate the package"
+
 TEMP_RESULT_FILE=$(ls ../*.deb)
+if [ "$TEMP_RESULT_FILE" = "" ]; then
+    cleanup "failed to find debian file"
+fi
+
+for fl in ${TEMP_RESULT_FILE}; do
+    destfile=$(basename ${fl} | sed 's/_\([0-9.]*_\)/_/')
```

Comment:
this is the change.
![image](https://user-images.githubusercontent.com/3674760/233921229-9deccc25-70e4-4d73-a21d-38b379cad017.png)